README.md

```diff
@@ -78,6 +81,22 @@ The domain should be a Guardian-owned domain. For internal tools,
 Name of the artifact containing the static resources. Should be uploaded in
 an earlier workflow step.
 
+## How to decommission a static site that is no longer needed
+
+You will need to ask DevX to do the first two steps, but you can do the third yourself.
+
+1. **Remove the stack**
+   Delete the `deploy-PROD-<app>` stack.
+   _Optional:_ You may also delete the relevant content from the S3 bucket, but this is usually unnecessary due to the small file sizes.
```

Comment:
I think the first one isn't a DevX exclusive thing - we just need access to the deployTools account. We could rephrase this to say "either get permissions from Janus, or ask DevX".